### PR TITLE
fix: add a line specifing about the visibility of menu based on permissions

### LINKED ISSUE #14863: Clarify who can change project visibility
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/changing-project-board-visibility

### What part(s) of the article would you like to see updated?

We have feedback that:

> Underneath the ... menu, there is no 'Settings' option. There is only Edit, Copy and View Archive. How do I change the visibility of my project board?

While a user doesn't have the right permissions (Org owner or project admin), they won't see settings. While we mention it in the permissions statement at the top of the article, this is really easy to miss. 

We should add another sentence to step 3 that you won't see this if you are not an org owner or project admin.

### Additional information

**NOTE** This issue is reserved for our MLH fellowship program.

### PR DESCRIPTION
**Pull request by riteshsp2000**
### Why:

Closes #14863 

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@riteshsp2000 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: data/reusables/project-management/project-board-visibility.md [hunk 1]
```diff
@@ -1 +1 @@
-By default, user-owned and organization-wide project boards are private and only visible to people with read, write, or admin permissions to the project board. {% ifversion ghae %}An internal{% else %}A public{% endif %} project board is visible to {% ifversion ghae %}any enterprise members{% else %}anyone{% endif %} with the project board's URL. Repository-level project boards share the visibility of their repository. That is, a private repository will have a private board, and this visibility cannot be changed.
+By default, user-owned and organization-wide project boards are private and only visible to people with read, write, or admin permissions to the project board. {% ifversion ghae %}An internal{% else %}A public{% endif %} project board is visible to {% ifversion ghae %}any enterprise members{% else %}anyone{% endif %} with the project board's URL. Repository-level project boards share the visibility of their repository and this visibility cannot be changed. That is, a private repository will have a private board.
```
Comment (lecoursen) [inline data/reusables/project-management/project-board-visibility.md:1]:
```suggestion
By default, user-owned and organization-wide project boards are private and only visible to people with read, write, or admin permissions to the project board. {% ifversion ghae %}An internal{% else %}A public{% endif %} project board is visible to {% ifversion ghae %}any enterprise members{% else %}anyone{% endif %} with the project board's URL. Repository-level project boards share the visibility of their repository, and this visibility cannot be changed. That is, a private repository will have a private board.
```

File: data/reusables/project-management/access-collaboration-settings.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
-1. Click {% octicon "kebab-horizontal" aria-label="The horizontal kebab icon" %}, then click **Settings**.
-![Settings option in drop-down menu from project board sidebar](/assets/images/help/projects/project-board-settings.png)
+1. Click {% octicon "kebab-horizontal" aria-label="The horizontal kebab icon" %}, then click **Settings**. You won't see this if you are not an org owner or project admin.
```
Comment (lecoursen) [inline data/reusables/project-management/access-collaboration-settings.md:1]: We shouldn't be introducing permissions information in the middle of a procedure like this. Instead, we should use permission statements at the beginning of each procedure that requires this step. 

For more information about permissions statements, see [the content models](https://github.com/github/docs/blob/main/contributing/content-model.md#permissions-statements). You should be able to search the repository for all uses of this reusable to find all the procedures that need these permissions statements.

**Review by lecoursen — CHANGES_REQUESTED** [review]

File: data/reusables/project-management/access-collaboration-settings.md (hunk 2 above)
Comment (riteshsp2000) [reply data/reusables/project-management/access-collaboration-settings.md]: Hey @lecoursen as per your suggestion, I shifted the permission statement at the start of the procedure.

File: content/issues/organizing-your-work-with-project-boards/managing-project-boards/changing-project-board-visibility.md [hunk 3]
```diff
@@ -24,6 +24,8 @@ shortTitle: Change visibility
 
 {% endtip %}
 
+Only organization members with *owner* or *project admin* roles can change project board visibility.
```
Comment (lecoursen) [inline content/issues/organizing-your-work-with-project-boards/managing-project-boards/changing-project-board-visibility.md:27]: Thanks @riteshsp2000! Let's tweak the phrasing as below, and also move it to a `permissions:` front matter:

- https://github.com/github/docs/tree/main/content#permissions
- https://github.com/github/docs/blob/main/contributing/content-model.md#permissions-statements

>Organization owners and people with admin access for a project board can change a project board's visibility.

**Comment by Amer1979amer** [comment]
> ### Why:
> Closes #14863
> 
> ### What's being changed:
> ### Check off the following:
> * [x]  I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
> * [x]  For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).
> 
> ### Writer impact (This section is for GitHub staff members only):
> * [ ]  This pull request impacts the contribution experience
>   
>   * [ ]  I have added the 'writer impact' label
>   * [ ]  I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

**Comment by janiceilene** [comment]
👋 @riteshsp2000 I see one [review comment](https://github.com/github/docs/pull/15455#discussion_r852118951) still open, has that one been resolved? If you resolved it, can you hit the resolve button and then ping me in here to let me know? At that point, I can put it back up for review, thanks!

**Review by ardaumbu330 — COMMENTED** [review]
Yes

File: content/issues/organizing-your-work-with-project-boards/managing-project-boards/changing-project-board-visibility.md (hunk 3 above)
Comment (AfirstS) [reply content/issues/organizing-your-work-with-project-boards/managing-project-boards/changing-project-board-visibility.md]: #

**Comment by janiceilene** [comment]
@riteshsp2000 Hope you're doing well! It looks like there hasn't been any new work on this PR for a while, so I'm going to close it now. Let me know if you want to pick it back up and we can reopen it!

**Comment by badbihh48** [comment]
unlink linked manual

**Pull request closed without merging**